R/pkg/R/DataFrame.R

```diff
@@ -2173,20 +2182,22 @@ setMethod("except",
             dataFrame(excepted)
           })
 
-#' Save the contents of the SparkDataFrame to a data source
+#' Save the contents of DataFrame to a data source.
 #'
-#' The data source is specified by the `source` and a set of options (...).
-#' If `source` is not specified, the default data source configured by
-#' spark.sql.sources.default will be used.
+#' Save the contents of the SparkDataFrame to a data source. The data source is specified by the
+#' `source` and a set of options (...). If `source` is not specified, the default data source
```

Comment:
hmm,
```The data source is specified by the
 `source` and a set of options (...).
```
that doesn't sound quite right? Data source should be in the source parameter.